src/main/java/core/mvc/HandlerAdapter.java

```diff
@@ -3,25 +3,10 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-public abstract class HandlerAdapter<T> {
+public interface HandlerAdapter {
 	
-	private final Class<T> parameterizedType;
 	
-	public HandlerAdapter(Class<T> parameterizedType) {
-		this.parameterizedType = parameterizedType;
-	}
+	public boolean supports(Object handler);
```

Comment:
```suggestion
	boolean supports(Object handler);
```
인터페이스의 메서드는 기본으로 public이기 때문에 코드에 명시하지 않아도 됩니다.

아래 첨부드린 글에 인터페이스의 메서드, 필드의 제어자에 대해 잘 정리되어 있으니 참고해보셔요.
[stackoverflow - Are “public” and “public final” redundant for interface methods?](https://stackoverflow.com/questions/9614708/why-should-we-declare-interface-methods-as-public)